bluepyopt/electrical/parameters.py

```diff
@@ -93,6 +93,10 @@ def check_bounds(self):
                     (self.name, self._value, str(self.lower_bound),
                      str(self.upper_bound)))
 
+    def destroy(self):
```

Comment:
Nope, this shouldn't be here.
'Parameters' still has to move a level up. 'destroy() / initiate()' will be something specific to Neuron (and maybe other simulators). But you can already replace this Parameter with NrnParameter in the mean time if you want.